Makefile

```diff
@@ -101,18 +101,24 @@ KEYSTONE_KUTTL_DIR       ?= ${OPERATOR_BASE_DIR}/keystone-operator/tests/kuttl/t
 KEYSTONE_KUTTL_NAMESPACE ?= keystone-kuttl-tests
 
 # Mariadb
-MARIADB_IMG         ?= quay.io/openstack-k8s-operators/mariadb-operator-index:latest
-MARIADB_REPO        ?= https://github.com/openstack-k8s-operators/mariadb-operator.git
-MARIADB_BRANCH      ?= main
+MARIADB_IMG             ?= quay.io/openstack-k8s-operators/mariadb-operator-index:latest
+MARIADB_REPO            ?= https://github.com/openstack-k8s-operators/mariadb-operator.git
+MARIADB_BRANCH          ?= main
 ifeq ($(DBSERVICE), galera)
-MARIADB             ?= config/samples/mariadb_v1beta1_galera.yaml
+MARIADB                 ?= config/samples/mariadb_v1beta1_galera.yaml
 else
-MARIADB             ?= config/samples/mariadb_v1beta1_mariadb.yaml
+MARIADB                 ?= config/samples/mariadb_v1beta1_mariadb.yaml
+endif
+MARIADB_CR              ?= ${OPERATOR_BASE_DIR}/mariadb-operator/${MARIADB}
+MARIADB_DEPL_IMG        ?= unused
+MARIADB_KUTTL_CONF      ?= ${OPERATOR_BASE_DIR}/mariadb-operator/kuttl-test.yaml
+MARIADB_KUTTL_DIR       ?= ${OPERATOR_BASE_DIR}/mariadb-operator/tests/kuttl/tests
+MARIADB_KUTTL_NAMESPACE ?= mariadb-kuttl-tests
+ifeq ($(DBSERVICE), galera)
+MARIADB_ASSERT          ?= ${MARIADB_KUTTL_DIR}/galera_deploy/01-assert.yaml
+else
+MARIADB_ASSERT          ?= ${MARIADB_KUTTL_DIR}/mariadb_deploy/01-assert.yaml
```

Comment:
Is `MARIADB_ASSERT` actually used anywhere?

Reply:
No it is no longer needed. I've removed it.